EventFiltering/PWGLF/filterf1proton.cxx

```diff
@@ -36,6 +36,7 @@
 #include "Common/DataModel/Multiplicity.h"
 #include "Common/DataModel/PIDResponse.h"
 #include "PWGLF/DataModel/LFStrangenessTables.h"
+#include "PWGHF/Core/PDG.h"
```

Comment:
Do we need the dependence from a PWGHF header for the PDG masses? I guess this should be moved to a more central location @vkucera

Reply:
Hi @mpuccio , yes, the plan is to move the masses to O2 eventually (see the timeline in the linked issue). I would avoid moving the header around for now at least until we get rid of `getMassPDG` completely and we can replace the header in one go in bulk.